modules/packages/apt_get

```diff
@@ -198,9 +197,16 @@ def one_package_argument(name, arch, version, is_apt_install):
         version_suffix = "=" + version
 
     if archs:
-        args = [name + ':' + arch + version_suffix for arch in archs]
+        for cur_arch in archs:
+            if cur_arch == platform_arch:
+                if is_apt_install:
+                    args.append((name + version_suffix, name + version_suffix))
+                else:
+                    args.append((name + version_suffix, name + ':' + cur_arch + version_suffix))
+            else:
+                args.append((name + ':' + cur_arch + version_suffix, name + ':' + cur_arch + version_suffix))
```

Comment:
A comment would be useful here as well, to explain why we add the two different variants.